src/hooks/useActiveOnboard.ts

```diff
@@ -14,125 +9,117 @@ import web3Utils from 'web3-utils'
-  const chainId = parseInt(chainIdHex)
-  const isSupported = Object.values(SupportedChainId).includes(chainId)
-  const supportedChainHex = isSupported ? chainIdHex : '0x1'
-  return {isSupported: isSupported, chainId: supportedChainHex}
+export function IsSupportedChain(chainIdHex: string): IsSupportedChainId {
+    const chainId = parseInt(chainIdHex)
+    const isSupported = Object.values(SupportedChainId).includes(chainId)
+    const supportedChainHex = isSupported ? chainIdHex : '0xa4ec'
+    return { isSupported: isSupported, chainId: supportedChainHex }
 }
 
-export function onboardContext(wstate: WalletState[]):ActiveOnboardInterface {
-  const [{ provider, label, accounts, chains }] = wstate
-  const web3provider = new Web3Provider(provider)
-  const chainIdHex = chains[0].id
-  const {isSupported, chainId} = IsSupportedChain(chainIdHex)
-  const error = !isSupported ? new UnsupportedChainId(chainIdHex) : undefined
-
-  const newContext = {
-    active: true,
-    accounts: accounts,
-    chainId: parseInt(chainId),
-    account: web3Utils.toChecksumAddress(accounts[0].address),
-    label: label,
-    eipProvider: provider as EIP1193ProviderExtended, 
-    library: web3provider,
-    error: error
-
-  }
-  return newContext
+export function onboardContext(wstate: WalletState[]): ActiveOnboardInterface {
+    const [{ provider, label, accounts, chains }] = wstate
+    const web3provider = new Web3Provider(provider)
+    const chainIdHex = chains[0].id
+    const { isSupported, chainId } = IsSupportedChain(chainIdHex)
+    const error = !isSupported ? new UnsupportedChainId(chainIdHex) : undefined
+
+    const newContext = {
+        active: true,
+        accounts: accounts,
+        chainId: parseInt(chainId),
+        account: web3Utils.toChecksumAddress(accounts[0].address),
+        label: label,
+        eipProvider: provider as EIP1193ProviderExtended,
+        library: web3provider,
+        error: error,
+    }
+    return newContext
 }
 
-export function useActiveOnboard<T = any>():ActiveOnboardInterface<T> {
-  const connectedWallets = useWallets()
-  const context = useMemo<ActiveOnboardInterface<Web3Provider>>(() => {
-    if (connectedWallets.length > 0) {
-      // console.log('update ActiveOnboard context')
-      const newContext = onboardContext(connectedWallets)
-      return newContext
-    } else {
-      return ({active: false, chainId: 122})
-    }
-  }, [connectedWallets])
+export function useActiveOnboard<T = any>(): ActiveOnboardInterface<T> {
+    const connectedWallets = useWallets()
+    const context = useMemo<ActiveOnboardInterface<Web3Provider>>(() => {
+        if (connectedWallets.length > 0) {
+            const newContext = onboardContext(connectedWallets)
+            return newContext
+        } else {
+            return { active: false, chainId: 42220 }
+        }
+    }, [connectedWallets])
 
-  return context
+    return context
```

Comment:
look above

Reply:
yes, unnecessary definition of const before return